ext/financialacls/tests/phpunit/BuildAmountHookTest.php

```diff
@@ -0,0 +1 @@
+<?php
```

Comment:
@eileenmcnaughton looks like this snuck in by accident

Reply:
dang

Reply:
ok fixed

Reply:
Still there for me

Reply:
second time lucky..

Reply:
@seamuslee001 it has passed now